components/ws-manager-bridge-api/go/BUILD.yaml

```diff
@@ -0,0 +1,12 @@
+packages:
+  - name: lib
+    type: go
+    deps:
+      - components/common-go:lib
```

Comment:
not sure we need the common-go dependency here, considering the package is just protoc generated code